compiler/rustc_lint/src/lib.rs

```diff
@@ -116,7 +116,6 @@ use types::*;
 use unit_bindings::*;
 use unused::*;
 
-/// Useful for other parts of the compiler / Clippy.
```

Comment:
This does seem like a valuable comment being lost.

Reply:
I disagree. That exact comment could be attached to many re-exports. Why would you re-export something if not because it's useful elsewhere in the compiler or other tools?